docs/interacting-with-openshift.md

```diff
@@ -83,7 +84,16 @@ $ minishift console
 To access a service exposed with a node port, run this command in a shell after starting Minishift to get the address:
 
 ```shell
-$ minishift service [-n NAMESPACE] [--url] NAME
+$ minishift openshift service [-n NAMESPACE] [--url] NAME
+```
+
+<a name="logs"></a>
+### Logs
+
+To access running OpenShift container logs, run this command in a shell after starting Minishift to get the logs:
+
```

Comment:
To access running OpenShift container logs, run this command in a shell after starting Minishift to get the logs: > To access logs for a running OpenShift container, use this command in a shell after starting Minishift: